src/webview/css/css.js

```diff
@@ -108,6 +108,16 @@ hr {
 .message-brief {
   padding: 0 1em 1em 4em;
 }
+.message-brief > .content > :first-child {
+  clear: right;
+}
+.message-brief > .content > :last-child::after {
+  content: var(--time);
+  float: right;
+  color: #999;
+  font-size: 0.8em;
```

Comment:
Not a good idea visually to have the sizes of the timestamps being different. Either use 0.9 as is the timestamp in 'message-full' or change both to 0.9

Reply:
That makes sense, thanks, I have changed it!